lib/std/os.zig

```diff
@@ -7056,3 +7056,51 @@ pub fn timerfd_gettime(fd: i32) TimerFdGetError!linux.itimerspec {
         else => |err| return unexpectedErrno(err),
     };
 }
+
+/// Whether or not the current target support SIGPIPE
+pub const have_sigpipe_support = switch (builtin.os.tag) {
```

Comment:
I think it would be worthwhile to make this an exhaustive switch so that people get compiler errors here when adding a new target OS and can't forget to add it to this list as easily.

Reply:
Whoa there are alot of os tags.  I've attempted to use the existence of `SIG.PIPE` instead to see if that would work.